bin/prepare_environment.sh

```diff
@@ -61,22 +66,23 @@ echo
 read -p "Enter your email: " email
 echo # newline
 
-## Save username and password as environment variables
-
-echo "$username" > /tmp/user_data
-echo "$password" >> /tmp/user_data
-echo "$email" >> /tmp/user_data
-
-if ! npm list -g bcrypt; then
-    npm install -g bcrypt
-fi
-
-hashed_password=$(node -e "console.log(require('bcrypt').hashSync(process.argv[1], 10));" "$password")
-escaped_hashed_password=$(printf '%s\n' "$hashed_password" | sed -e 's/[\/&]/\\&/g')
+# Hash the password
+docker pull epicsoft/bcrypt > /dev/null 2>&1
+encrypted_password=$(docker run --rm epicsoft/bcrypt hash "$password" 10 | sed -e 's/[\/&]/\\&/g')
+docker rmi epicsoft/bcrypt > /dev/null 2>&1
 
 ## Replace the example_user and example_pass strings with the new user and password
+## Due to differences in GNU and Mac implementations, we don't do it in place
+
+function setVariables() {
```

Comment:
This function creates a `settings.js-e` because of the `-i` flag. Remove this flag and it will probably work.

Reply:
Ooops, good catch! 🚀